### PR TITLE
Support PHP 8.2

### PR DESCRIPTION
**Pull request by heyanlong**
I noticed that PHP 8.2 was released on December 08, 2022. So add it in
https://www.php.net/archive/2022.php#2022-12-08-1

### CODE REVIEW

**Comment by jmjoy** [comment]
Wait `phper` to release new version.

**Comment by jmjoy** [comment]
`phper` has updated to 0.8.0.

https://github.com/phper-framework/phper/releases/tag/0.8.0

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -62,6 +62,8 @@ jobs:
             swoole: "5.0.0"
           - php: "8.1"
             swoole: "5.0.0"
+          - php: "8.2"
+             swoole: "5.0.0"
```
Comment (wu-sheng) [inline .github/workflows/rust.yml:66]:
```suggestion
          - php: "8.2"
            swoole: "5.0.0"
```

**Comment by jmjoy** [comment]
@heyanlong Remove your company email in commit message?

**Pull request merged**